examples/github-with-qemu.yml

```
@@ -25,7 +25,7 @@ jobs:
         env:
           # configure cibuildwheel to build native archs ('auto'), and some
           # emulated ones
```

```suggestion
          # configure cibuildwheel on Linux to build native archs ('auto'),
          # and to split the remaining architectures between the x86_64 and
          # ARM runners
```

I don't love this way of doing ternary expressions, but it's all we have in GHA workflows, so I think adding a comment is nice.

---

I included your proposal and extended a bit on the `auto armv7l` peculiarity. That might be a bit too verbose but that's strange enough to deserve a also comment (it's "native" in some circumstances hence not in `auto`).

---

This change looks good to me. Question for another PR, but it makes me wonder if we could detect armv7l support and include that in `auto` on machines where that's possible.

---

> if we could detect armv7l support and include that in auto on machines where that's possible.

That might be possible:
- don't allow on platform other that Linux
- try to run a small statically linked Linux exe & check if it works.